src/components/schemas/report.yaml

```diff
@@ -488,6 +488,25 @@ report_action_with_expenses_ids_to_eject:
           - $ref: './fields.yaml#/id_string'
       example: [ 'txelntgwgwll' ]
 
+add_approvers_bulk_in:
+  type: object
+  additionalProperties: False
+  required:
+    - id
+    - approver_emails
+  properties:
+    id:
+      type: string
+      description: |
+        id of report, for which approvers need to be added
```

Comment:
`id of report, for which approvers need to be added` --> `id of the report, to which approvers need to be added`